### DISCUSSION TITLE
Want to Add Step-by-step Guide for contribuition in contribuiting.md file

### DISCUSSION THREAD

**Discussion by taqui-786** [renovatebot/renovate#24977] [Suggest an Idea]
### Tell us more.

I **propose** adding a step-by-step contribution guide to contribuition.md. This guide will streamline onboarding for new contributors, enhance collaboration, and ensure consistent contributions. It will serve as a valuable resource, fostering a more inclusive and productive open-source community.

**Please Assign me**

**Comment by rarkins** [comment]
Great, please go ahead! It might be easier for it to be a separate markdown file though?
